ruby/vampires.rb

```diff
@@ -0,0 +1,62 @@
+puts "Enter number of employees"
+no_of_employee = gets.chomp
+
+i = 1
+
+# Starting a loop
+while i <= no_of_employee.to_i
+
+  # Getting name from the prospective new employee
+  puts "What is your name?"
+  employee_name = gets.chomp
+
+  # Getting employee's age and year of birth (Age should be less than 100)
+  puts "Please provide your age."
+  age = gets.to_i
+
+  puts "What is your year of birth?"
+  year = gets.to_i
+
+  current_year = 2017
```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.